app/editor/menus/notice.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  DoneIcon,
+  ExpandedIcon,
+  InfoIcon,
+  StarredIcon,
+  WarningIcon,
+} from "outline-icons";
+import { EditorState } from "prosemirror-state";
+import * as React from "react";
+import { NOTICE_TYPES } from "@shared/editor/nodes/Notice";
+import { MenuItem } from "@shared/editor/types";
+import { Dictionary } from "~/hooks/useDictionary";
+
+export default function noticeMenuItems(
+  state: EditorState,
+  readOnly: boolean | undefined,
+  dictionary: Dictionary
+): MenuItem[] {
+  const { selection } = state;
+  const { $from } = selection;
+  const node = $from.node(-1);
+
+  const currentStyle = node.attrs.style;
+
+  const canView = (type: NOTICE_TYPES) => {
+    if (readOnly) {
+      return false;
+    }
+
+    return currentStyle !== type;
+  };
+
+  return [
+    {
+      name: "changeNoticeType",
```

Comment:
```suggestion
      name: "container_notice",
```

This can be named the same as the node type, no need for the "fake" extra command then

Reply:
I tried doing like this but it's not working, the menu doesn't appear

Reply:
It did the work! Thanks